src/using-workflow/using-docker-images.md

```diff
@@ -33,13 +33,9 @@ In order to deploy Docker images, they must conform to the following requirement
 Use `deis create` to create an application on the [controller][].
 
     $ mkdir -p /tmp/example-go && cd /tmp/example-go
-    $ deis create
+    $ deis create example-go --no-remote
     Creating application... done, created example-go
 
-!!! note
```

Comment:
I'd keep this note in since the rest of the commands don't use `--app` to specify the application name.